internal/indexnode/param_table.go

```diff
@@ -136,6 +138,14 @@ func (pt *ParamTable) initMetaRootPath() {
 	pt.MetaRootPath = path.Join(rootPath, subPath)
 }
 
+func (pt *ParamTable) initIndexRootPath() {
+	rootPath, err := pt.Load("etcd.rootPath")
```

Comment:
@xiaocai2333 @xiaofan-luan  do we need a config item named minio.rootPath ?

Reply:
'rootPath' has been added for minio.